src/javascripts/ng-admin/es6/lib/Entity/Entity.js

```diff
@@ -13,6 +14,7 @@ import ExportView from '../View/ExportView';
 class Entity {
     constructor(name) {
         this._name = name;
+        this._dataStore = new DataStore(name);
```

Comment:
I don't think entities should have a datastore. Entities should be immutable (and reusable) once the configuration phase is over. Putting the DataStore there prevents this and mixes up roles.